app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Here is a list of users</h1>
+<ul>
+  <% @users.each do |user| %>
```

Comment:
[OPTIONAL]
- Instead of using `@objects.each`, you can create a partial for example `app/views/users/\_user.html.erb` and render the users here:
```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```